Islandora/islandora_newspaper_batch
# ISLANDORA-2030: Centralize mods_to_dc.xsl

**JIRA Ticket**: https://jira.duraspace.org/browse/ISLANDORA-2030

* Other Relevant Links (Google Groups discussion, related pull requests, Release pull requests, etc.)

Companion pull requests on islandora_batch, islandora_newspaper_batch, islandora_book_batch, islandora_importer

# What does this Pull Request do?

Puts mods_to_dc.xsl in a single place: Islandora XML Forms. This simplifies updates to the mods_to_dc.xsl file when Library of Congress makes changes, and paves the way for further centralization and better modular functions.

# What's new?

References to mods_to_dc.xsl now point at islandora_xml_forms module, and a dependency has been added.

# How should this be tested?

Create a new batch ingest, verify that the DC datastream is still generated as expected.

# Interested parties
@Islandora/7-x-1-x-committers
